ceph-deploy/build/build

```diff
@@ -120,40 +137,11 @@ then
         exit 2
     fi
 
-    # Build Repo
-    PKG=../ceph-deploy*.changes
-    mkdir -p $REPO/conf
-    if [ -e $REPO/conf/distributions ] ; then
-        rm -f $REPO/conf/distributions
-    fi
+    [ "$FORCE" = true ] && chacra_flags="--force" || chacra_flags=""
+
+    # push binaries to chacra
+    find ../ | egrep "*\.(changes|deb|dsc|gz)$" | egrep -v "(Packages|Sources|Contents)" | chacractl binary ${chacra_flags} create ${chacra_endpoint}
```

Comment:
could this be a path determined from `$WORKSPACE` vs. using something relative like `../` ? I know I've spent a lot of times trying to figure out the exact location before and normalizing in using `$WORKSPACE`